sky/clouds/service_catalog/data_fetchers/fetch_aws.py

```diff
@@ -13,23 +13,27 @@
 
 from sky.adaptors import aws
 
+# Turn off the regions disabled for a new AWS account by default.
 ALL_REGIONS = [
     'us-east-1',
     'us-east-2',
     'us-west-1',
     'us-west-2',
     'ca-central-1',
     'eu-central-1',
+    # 'eu-central-2', # disabled
     'eu-west-1',
     'eu-west-2',
-    'eu-south-1',
+    # 'eu-south-1',
+    # 'eu-south-2', # disabled
     'eu-west-3',
     'eu-north-1',
-    'me-south-1',
+    # 'me-south-1',
     # 'me-central-1', # failed for no credential
-    'af-south-1',
-    'ap-east-1',
-    'ap-southeast-3',
+    # 'af-south-1',
+    # 'af-south-2', # disabled
```

Comment:
Q: does this comment apply to L33-34, and L37's comment apply to L35-37?

Patterns we have used:
```
    # 'af-south-1',  # apply to this line
    # 'af-south-2',  # another per-line comment
```
or

```
    # disabled (apply to one block):
    # 'af-south-1', 
    # 'af-south-2',  
    # another block:
    # 'xx-south-1', 
    # 'xx-south-2',  
```